database/mock.go

```diff
@@ -14,23 +14,26 @@
 
 package database
 
-import "time"
+import (
+	"github.com/coreos/clair/services"
+	"time"
+)
 
 // MockDatastore implements Datastore and enables overriding each available method.
 // The default behavior of each method is to simply panic.
 type MockDatastore struct {
```

Comment:
We should probably break this down into a few different MockXxxService types to parallel the refactor.